### PR TITLE
Add codely as supporter sponsor

### PR DESCRIPTION
**Pull request by alexcuesta**
https://trello.com/c/PYJktIwx

### CODE REVIEW

File: 2022/_data/sponsors.yml [hunk 1]
```diff
@@ -114,3 +114,9 @@ supporters:
     image:
       src: "/assets/img/sponsors/jetbrains.png"
       alt: JetBrains
+
+  - name: CodelyTV
+    href: https://codely.tv
```
Comment (icougil) [inline 2022/_data/sponsors.yml:119]:
```suggestion
    href: https://codely.com/pro/jbcnconf22
```

File: 2022/_data/sponsors.yml [hunk 2]
```diff
@@ -114,3 +114,9 @@ supporters:
...
+    image:
+      src: "/assets/img/sponsors/codely.png"
```
Comment (icougil) [inline 2022/_data/sponsors.yml:121]: can we try to remove its transparent background ?
you can try for example: https://www.remove.bg/es or https://www.adobe.com/express/feature/image/remove-background

Comment (alexcuesta) [reply 2022/_data/sponsors.yml:121]: Sure! I did not know it was a bad thing. Cheers

**Comment by alexcuesta** [comment]
I am closing this PR. I forgot to do it in a separate branch!

**Pull request closed without merging**